templates/home/reports/pre-a11y-1-fr.html

```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Pre-accessibility assessment #1",
+	"language": "fr",
+	"description": "Assessing the main content of the GC Home Page template",
```

Comment:
Translate the title and description in French, use google translate.

Reply:
https://github.com/AtlakM/GCWeb/tree/wet-386-pre-assessment-gc-home-page

Reply:
fixed